### DISCUSSION TITLE
How to change from MiB or GiB to MB and GB

### DISCUSSION THREAD

**Discussion by Joao-Schio** [fastfetch-cli/fastfetch#1014] [Q&A] [answered]
Hello there, I was wondering how to change the unit of the memory and disk modules from MiB and GiB to MB and GB

**Comment by CarterLi** [comment]
`fastfetch -h binary-prefix`

**Reply by onefish2** [reply]
Hello,

Can you provide the details on what to add to the schema portion of the config file so I can see MB and GB instead of MiB and GiB?

**Reply by onefish2** [reply]
No need for "stat": true. That printed a bunch of info on the right side of the screen.

I just used "binaryPrefix": "jedec" and that worked.

Thank you!!

**Comment by CarterLi** [comment]
Can you try this?

![image](https://github.com/user-attachments/assets/a553d820-2822-4ee2-979f-24d2e525a8d1)

**Reply by Joao-Schio** [reply]
Since this is the thread that is in the readme, do you mind adding the updated version of this config ?

**Comment by JohRest** [comment, marked as answer]
To change the units from MiB/GiB to MB/GB in Fastfetch, you’ll want to adjust the binaryPrefix setting in your configuration. Here's how you can do it:

🛠️ Update Fastfetch Config to Use MB/GB Units

In your Fastfetch JSON config file (usually located at ~/.config/fastfetch/config.jsonc), find or add the following under the display section:

`json
"display": {
  "size": {
    "binaryPrefix": "jedec"
  }
}
`

🔍 What This Does
- Setting "binaryPrefix": "jedec" switches the unit system from IEC (MiB/GiB) to JEDEC (MB/GB).
- This affects how memory and disk sizes are displayed—using base 10 (1 GB = 1000 MB) instead of base 2 (1 GiB = 1024 MiB).

You can also pass this directly via command line if you prefer:

`bash
fastfetch --size-binary-prefix jedec
`

**Reply by Joao-Schio** [reply]
Thank you
